src/libp2p_ipc/libp2p_ipc.capnp

```diff
@@ -98,282 +73,319 @@ struct ValidationMessage {
+      setGatingConfig @2 :Libp2pHelperInterface.SetGatingConfig.Request;
+      listen @3 :Libp2pHelperInterface.Listen.Request;
+      getListeningAddrs @4 :Libp2pHelperInterface.GetListeningAddrs.Request;
+      beginAdvertising @5 :Libp2pHelperInterface.BeginAdvertising.Request;
+      addPeer @6 :Libp2pHelperInterface.AddPeer.Request;
+      listPeers @7 :Libp2pHelperInterface.ListPeers.Request;
+      generateKeypair @8 :Libp2pHelperInterface.GenerateKeypair.Request;
+      publish @9 :Libp2pHelperInterface.Publish.Request;
+      subscribe @10 :Libp2pHelperInterface.Subscribe.Request;
+      unsubscribe @11 :Libp2pHelperInterface.Unsubscribe.Request;
+      addStreamHandler @12 :Libp2pHelperInterface.AddStreamHandler.Request;
+      removeStreamHandler @13 :Libp2pHelperInterface.RemoveStreamHandler.Request;
+      openStream @14 :Libp2pHelperInterface.OpenStream.Request;
+      closeStream @15 :Libp2pHelperInterface.CloseStream.Request;
+      resetStream @16 :Libp2pHelperInterface.ResetStream.Request;
+      sendStream @17 :Libp2pHelperInterface.SendStream.Request;
+      setNodeStatus @18 :Libp2pHelperInterface.SetNodeStatus.Request;
+      getPeerNodeStatus @19 :Libp2pHelperInterface.GetPeerNodeStatus.Request;
+    }
+  }
+
+  struct RpcResponseSuccess {
+    union {
+      configure @0 :Libp2pHelperInterface.Configure.Response;
+      setGatingConfig @1 :Libp2pHelperInterface.SetGatingConfig.Response;
+      listen @2 :Libp2pHelperInterface.Listen.Response;
+      getListeningAddrs @3 :Libp2pHelperInterface.GetListeningAddrs.Response;
+      beginAdvertising @4 :Libp2pHelperInterface.BeginAdvertising.Response;
+      addPeer @5 :Libp2pHelperInterface.AddPeer.Response;
+      listPeers @6 :Libp2pHelperInterface.ListPeers.Response;
+      generateKeypair @7 :Libp2pHelperInterface.GenerateKeypair.Response;
+      publish @8 :Libp2pHelperInterface.Publish.Response;
+      subscribe @9 :Libp2pHelperInterface.Subscribe.Response;
+      unsubscribe @10 :Libp2pHelperInterface.Unsubscribe.Response;
+      addStreamHandler @11 :Libp2pHelperInterface.AddStreamHandler.Response;
+      removeStreamHandler @12 :Libp2pHelperInterface.RemoveStreamHandler.Response;
+      openStream @13 :Libp2pHelperInterface.OpenStream.Response;
+      closeStream @14 :Libp2pHelperInterface.CloseStream.Response;
+      resetStream @15 :Libp2pHelperInterface.ResetStream.Response;
+      sendStream @16 :Libp2pHelperInterface.SendStream.Response;
+      setNodeStatus @17 :Libp2pHelperInterface.SetNodeStatus.Response;
+      getPeerNodeStatus @18 :Libp2pHelperInterface.GetPeerNodeStatus.Response;
+    }
+  }
+
+  struct RpcResponse {
+    header @0 :RpcMessageHeader;
+    union {
+      error @1 :Text;
+      success @2 :RpcResponseSuccess;
+    }
+  }
+
+  struct PushMessage {
+    header @0 :PushMessageHeader;
+
+    # union {
+    validation @1 :Libp2pHelperInterface.Validation;
+    # }
+  }
+
+  struct IncomingMessage {
```

Comment:
Let's use just `Message` maybe?

Because it's outgoing for Go code and incoming for Ocaml.